### PR TITLE
add support XP-Pen Deco 02

### PR DESCRIPTION
**Pull request by PSandro**
I own a `XP-Pen Deco 02` and stumbled across this project. It's exactly what I was looking for! :tada: 

For adding Deco 02 support, the files from Deco 03 were copied and slightly modified. I had the issue that when pressing button 6, it would additionally count it as a dial event for REL_WHEEL left.

Here is the data dump for the actions:
# button 1
```
03 00 05 00 00 00 00 00
02 f0 01 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# button 2
```
03 00 2c 00 00 00 00 00
02 f0 02 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# button 3
```
03 00 08 00 00 00 00 00
02 f0 04 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# button 4
```
03 00 0c 00 00 00 00 00
02 f0 08 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# button 5
```
03 05 1d 00 00 00 00 00
02 f0 10 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# button 6
```
03 01 16 00 00 00 00 00
02 f0 20 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# dial left
```
03 01 2d 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

# dial right
```
03 01 2e 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
03 00 00 00 00 00 00 00
02 f0 00 00 00 00 00 00 00 00
```

### CODE REVIEW

**Comment by kurikaesu** [comment]
Thanks for the pull request!

Were you able to resolve the issue with button 6 in your branch?

If so, I'll go ahead and merge this in :)

File: src/deco_02.cpp [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    }
+}
+
+void deco_02::handleNonUnifiedDialEvent(libusb_device_handle *handle, unsigned char *data, size_t dataLen) {
+    if (data[1] == 0x01) {
+        // Take the dial
+        short dialValue = 0;
+
+        switch (data[2]) {
+            case 0x2e:
```
Comment (PSandro) [inline src/deco_02.cpp:109]: explicitly listen for dial event values

**Comment by PSandro** [comment]
yep, everything seems to be working just fine now :)

**Comment by kurikaesu** [comment]
Great! Merging in 👍

**Pull request merged**